server/concepts/Tournament.ts

```diff
@@ -123,6 +123,27 @@ export default class Tournament implements TournamentWithoutMS {
     /* eslint-enable @typescript-eslint/no-non-null-assertion */
   }
 
+  public static async getById(tournamenId: string, token: string) {
+    const tournament = await TournamentRepository.getById(tournamenId);
+    if (!tournament) return null;
+    const enrichedTournament = await Tournament.enrichWithMSUsers(tournament, token);
+    return enrichedTournament;
+  }
+
+  public static async getAll(page :number, pageSize: number, token: string) {
+    const data = await TournamentRepository.getAll(page, pageSize);
+    const enrichedTournaments = await Tournament.enrichTournamentsWithMSUsers(data.tournaments, token);
+
+    const enrichedData: {
+      totalRows: number,
+      tournaments: ITournament[],
+    } = {
+      totalRows: data.totalRows,
+      tournaments: enrichedTournaments,
+    };
+    return enrichedData;
```

Comment:
```suggestion
    return {
      totalRows: data.totalRows,
      tournaments: enrichedTournaments,
    };
```
More enjoyable, this return will infer the type from the objects